### PR TITLE
Nfdiv 1486 respondent eligible final order

### PR DESCRIPTION
**Pull request by mark-butler-solirius**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-1486

### Change description ###
Notify respondent that final order has not yet been applied for once the valid time has elapsed

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java [hunk 1]
```diff
@@ -157,6 +157,16 @@
...
 
+    @CCD(
+        label = "The applicant wants to continue with their application."
+    )
+    private YesOrNo continueApplication;
+
+    @CCD(
+        label = "Has the applicant been sent a reminder to apply for the Final Order?"
+    )
+    private YesOrNo finalOrderReminderSent;
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java:168]: should this be part of FinalOrder class ?

File: src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemNotifyRespondentApplyFinalOrderTask.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    String applicantCanApplyFromDate = (String) caseDataMap.getOrDefault(APP_ELIGIBLE_DATE, null);
+                    String applicationType = (String) caseDataMap.get(APPLICATION_TYPE);
+
+                    log.info("Found Case Id {} Application Type {} Eligible From {} Reminder Sent? {}",
+                        caseDetails.getId(),
+                        applicationType,
+                        applicantCanApplyFromDate,
+                        finalOrderReminderSent);
+
+                    if (respondentCanApplyFromDate == null) {
```
[the remark this answers is not in the document]
Comment (mark-butler-solirius) [reply src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemNotifyRespondentApplyFinalOrderTask.java:85]: Yep - that looks better

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java (hunk 1 above)
Comment (mark-butler-solirius) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java:168]: I added it to the Applicant class as I thought that a final order reminder could be sent to the Applicant and the respondent.

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java:168]: checked with Linus it should be in FinalOrder class

Comment (mark-butler-solirius) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java:168]: ok

Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java:168]: yeah, it'll need two, one for each applicant.

Comment (mark-butler-solirius) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java]: Updated name from finalOrderReminderSent to finalOrderReminderSentApplicant2 and added the ...Applicant1 field for when it is required

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**